__init__.py

```diff
@@ -8,18 +8,11 @@
 i18n_catalog = i18nCatalog("CuraSolidWorksIntegrationPlugin")
 
 if Platform.isWindows():
-    # For installation check
-    import winreg
-    # The reader plugin itself
-    from . import SolidWorksReader
+    from . import SolidWorksUtils
 
     def is_SolidWorks_available():
-        try:
-            # Could find a better key to detect whether SolidWorks is installed..
-            winreg.OpenKey(winreg.HKEY_CLASSES_ROOT, "SldWorks.Application")
-            return True
-        except:
-            return False
+        solidworks = SolidWorksUtils.getAvailableSolidWorksVersions()
+        return True if solidworks else False
```

Comment:
I will change the naming. `solidworks` is actually a list of solidworks it has found. Return True if there is anything found otherwise False.